cmd/juju/action/list_test.go

```diff
@@ -66,6 +66,11 @@ func (s *ListSuite) TestInit(c *gc.C) {
 		args:                 []string{"--format=yaml", "--schema", validServiceId},
 		expectedOutputSchema: true,
 		expectedSvc:          names.NewApplicationTag(validServiceId),
+	}, {
```

Comment:
We should probably also have a test for "--format default" being used. "default" will show up in --help so users will be tempted to type it.